prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -1561,4 +1562,53 @@ class SantaClaus : CoolTest() {
     private fun createBad(message: String): Boolean {
         return bad("***$name Test FAILED***: $message")
     }
+}
+
+class OtcProctored : CoolTest() {
+    override val name = "otcproctored"
+    override val description = "Verify that otc/proctored flags are working as expected on api response"
+    override val status = TestStatus.SMOKE
+    val failures = mutableListOf<String>()
+
+    override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        val otcMap = mapOf(
```

Comment:
A kotlin Pair object would be more appropriate instead of a map for `otcMap`, since you aren't really using the map as a map (eg, to look things up).

Reply:
I'll make this changes as part of next PR